framework/test/integration/specs/controller/ipc_channel.spec.ts

```diff
@@ -67,24 +67,24 @@ const beta = {
 	},
 };
 /* eslint-disable jest/no-disabled-tests */
-describe.skip('ChildProcessChannel', () => {
+describe.skip('IPCChannel', () => {
```

Comment:
skip?

Reply:
Yes skipped in recent PR, have another issue to look into why they are unstable.

Reply:
https://github.com/LiskHQ/lisk-sdk/issues/5512